prow/e2e-suite.sh

```diff
@@ -82,5 +82,5 @@ if ${SINGLE_MODE}; then
 
 else
     echo "Executing e2e test suite"
-    time ISTIO_DOCKER_HUB=$HUB E2E_ARGS="${E2E_ARGS[@]}" make e2e_all
+    time ISTIO_DOCKER_HUB=$HUB E2E_ARGS="${E2E_ARGS[@]}" make e2e_all > tee >(go-junit-report > junit.xml)
```

Comment:
Can you move this to the makefile, so it can be used in circleci too ?

Reply:
Done